spyder/utils/help/tutorial.rst

```diff
@@ -1,382 +1,324 @@
+*Spyder* is an Integrated Development Environment (IDE) for scientific computing, written in and for the Python programming language.
+It comes with an :guilabel:`Editor` to write code, a :guilabel:`Console` to evaluate it and view the results at any time, a :guilabel:`Variable Explorer` to examine the variables defined during evaluation, and several other facilities to help you effectively develop the programs you need as a scientist.
 
 
-This tutorial was originally authored by
-`Hans Fangohr <https://fangohr.github.io/>`__ from the
-University of Southampton (UK),
-and subsequently updated for Spyder 3.3.x by the development team
-(see `Historical note`_ for more details).
+This tutorial was originally authored by `Hans Fangohr <https://fangohr.github.io/>`__ from the University of Southampton (UK), and subsequently updated for Spyder 3.3.x by the development team (see `Historical note`_ for more details).
 
 
 
+=======================
 First steps with Spyder
-#######################
+=======================
+
+This section is aimed at Python and Spyder beginners.
+If you find it too simple, please continue to the `next one <python-beginners-ref_>`__.
 
-This section is aimed at Python and Spyder beginners. If you find it too
-simple, please continue to the `next one <python_beginners_ref_>`_.
 
 Execute a given program
------------------------
+~~~~~~~~~~~~~~~~~~~~~~~
 
-* We are going to use this program as a first example:
+We are going to use run program as a first example:
 
-  .. code-block:: python
+.. code-block:: python
 
-     # Demo file for Spyder Tutorial
-     # Hans Fangohr, University of Southampton, UK
+    # Demo file for Spyder Tutorial
+    # Hans Fangohr, University of Southampton, UK
 
-     def hello():
-         """Print "Hello World" and return None."""
-         print("Hello World")
+    def hello():
+        """Print "Hello World" and return None."""
+        print("Hello World")
 
-     # main program starts here
-     hello()
+    # Main program starts here
+    hello()
 
-* To use this program, please create a new file in the Spyder editor pane
-  (with ``File > New file``, or ``Ctrl+N``/``Cmd-N``).
-  Then copy and paste the code inside the box above into the file,
-  and save it with the name ``hello.py``.
+#. Please create a new file in the Spyder :guilabel:`Editor` pane (with :menuselection:`File --> New file`, or :kbd:`Ctrl-N`/ :kbd:`Command-N`).
+   Then copy and paste the code inside the box above into the file, and save it with the name :file:`hello.py`.
 
-* To execute the program, select ``Run > Run`` from the menu (or press F5), and
-  confirm the ``Run settings`` if required.
+#. To execute the program, select :menuselection:`Run --> Run` from the menu (or press :kbd:`F5`), and confirm the ``Run settings`` if required.
 
-  If this is your first time, you should see an output like this::
+#. If this is your first running something, you should see an output like this:
```

Comment:
"time" was deleted in this change - was that accidental?

Reply:
Yes, thanks.

Reply:
Fixed!